nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -1426,6 +1426,22 @@
                 sys.exit("Nuitka: For this environment to load, need to use this as an option to compile with '--include-module=%s'." % mod_name)
       when: 'not deployment'
 
+- module-name: 'h2o' # checksum: a246b3c9
+  data-files:
+    dirs:
+      - 'h2o_data/'
+    patterns:
+      - '*.txt'
+      - '*.csv'
+  dlls:
+    - from_filenames:
+        relative_path: 'backend/bin/'
```

Comment:
The trailing slash in these paths should not be, I need to figure out how to make auto-format remove them, or the schema to mark them as an error, because they all are supposed to be directory names.

Reply:
resolved.